src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -81,6 +99,39 @@ public Command parseCommand(String userInput) throws ParseException {
         case ClearByCommand.COMMAND_WORD:
             return new ClearByCommandParser().parse(arguments);
 
+        case FindTaskCommand.COMMAND_WORD:
+            return new FindTaskCommandParser().parse(arguments);
+
+        case TodoCommand.COMMAND_WORD:
+            return new TodoCommandParser().parse(arguments);
+
+        case ListTodoCommand.COMMAND_WORD:
+            return new ListTodoCommand();
+
+        case EditDeadlineCommand.COMMAND_WORD:
+            return new EditDeadlineCommandParser().parse(arguments);
+
+        case EditNoteContentCommand.COMMAND_WORD:
+            return new EditContentCommandParser().parse(arguments);
+
+        case DeleteTodoCommand.COMMAND_WORD:
+            return new DeleteTodoCommandParser().parse(arguments);
+
+        case ClearTodoCommand.COMMAND_WORD:
+            return new ClearTodoCommand();
+
+        case NoteCommand.COMMAND_WORD:
+            return new NoteCommandParser().parse(arguments);
+
+        case ListNoteCommand.COMMAND_WORD:
+            return new ListNoteCommand();
+
+        case DeleteNoteCommand.COMMAND_WORD:
+            return new DeleteNoteCommandParser().parse(arguments);
+
+        case ClearNoteCommand.COMMAND_WORD:
+            return new ClearNoteCommand();
+
```

Comment:
I feel like splitting Note and Todo related command into different keywords may be a little confusing to user 🤔